### PR TITLE
[Support/ LawsAndRegulations/ AboutPortal] - no info displaying

### PR DESCRIPTION
**Pull request by koliapatskal**
#1583 

* Added a app-no-result-card component to display a message

*Added new value NoResultTitle

*Removed css property that caused info to overlap breadcrumbs

### CODE REVIEW

**Comment by koliapatskal** [comment]
Display information before deleting css property.

image

File: src/app/shell/info/about/about.component.ts [hunk 1]
```diff
@@ -26,6 +27,7 @@ export class AboutComponent implements OnInit, OnDestroy {
 
   platformInformation: CompanyInformation;
 
+  readonly emptyList = NoResultsTitle.noInfo;
```
Comment (litvinets) [inline src/app/shell/info/about/about.component.ts:30]: this is not the proper naming, change it to more suitable such as noData/noInfo

Comment (litvinets) [inline src/app/shell/info/about/about.component.ts:30]: move read-only field to the very top (before platformInformation) so we will stay consistent we our code style. it's not a strict rule, just approach for our project

**Review by litvinets — CHANGES_REQUESTED** [review]
spmething is wrong with you branch, therfe should be an image before the content 
image
image

**Review by litvinets — APPROVED** [review]

**Pull request merged**